metrics/coverage_high_water_mark

```diff
@@ -1 +1 @@
-94.0200
\ No newline at end of file
+93.6600
```

Comment:
If you zoom down to the tests, I had to disable a few tests - we had code and tests supporting hint values for date/time values which don't actually exist in the spec.  Now that we're doing validation using the correct list of supported hint values, those tests can't be run.

Adding those hint values makes sense, for sure, though.  Adding those hint values and making sure all the different drivers support them correctly is likely to spiral just a bit from this already 56 file PR, so I'm proposing to comment out those tests and leave those hints as unsupported.  I have a high priority follow-up backlog task to handle these.  It's not normal at least in internal use to be passing in date/time hints, so I'm not terribly concerned about breaking any existing out-of-spec code with this.

Commented out tests:
* https://github.com/bluelabsio/records-mover/pull/66/files#diff-b9d3d8d84af11c791565c585049c18be
* https://github.com/bluelabsio/records-mover/pull/66/files#diff-4abbf34edf14477a63b4fe26b409b96e

Backlog task: https://app.asana.com/0/1128138765527694/1178155809546696